frameworks/cassandra/universe/marathon.json.mustache

```diff
@@ -153,7 +154,6 @@
     "CASSANDRA_DOCKER_IMAGE": "{{resource.assets.container.docker.cassandra}}",
     "JAVA_URI": "{{resource.assets.uris.jre-tar-gz}}",
     "TASKCFG_ALL_CASSANDRA_LOCATION_DATA_CENTER": "{{service.data_center}}",
-    "TASKCFG_ALL_CASSANDRA_LOCATION_RACK": "{{service.rack}}",
```

Comment:
Are there any downsides to removing this config value?  Just checking.

Reply:
Having an end-user set the "rack" string was useless from the beginning.  Everything was always on the same rack named the same thing.

Reply:
@triclambert spoke with the relevant authorities.  Removing this is fine.